### DISCUSSION TITLE
Addition of Miscellaneous Meeting Option

### DISCUSSION THREAD

**Discussion by AGNMM18** [CareTogether/CareTogetherCMS#408] [Ideas]
Enhancement request: Can we add an option along with +adult, +child, +upload document, and +notes for a meeting, either with the HF or the parent, and a home visit?  We discovered when entering a referral, there is no way to enter a home visit for an arrangement if the system won't prompt for one during the time the hosting is occurring.  (Example, Recurring Hosting (familiar HF)) - even if the system doesn't require one, we need to be able to note that it occurred, and would prefer the option for a visit as opposed to a general note.

**Comment by LarsKemmann** [comment]
Pick from available requirements that can be additionally completed

**Maybe just provide a ➕ button inline...?
